src/Task.ts

```diff
@@ -109,6 +109,10 @@ export class Task {
     // description: '#dog #car http://www/ddd#ere #house'
     // matches: #dog, #car, #house
     public static readonly hashTags = /(^|\s)#[^ !@#$%^&*(),.?":{}|<>]*/g;
+    public static readonly hashTagsFromEnd = new RegExp(
+        this.hashTags.source + '$',
```

Comment:
Nice - I didn't know it was possible to access the source. This might enable some of the error messages in Tasks to be improved.